examples/weathertabs/js/alt.js

```diff
@@ -0,0 +1,2 @@
+var Alt = require('../../../src/alt');
```

Comment:
yeah, I'll do that. When I was touching the src file I did not want to build alt each time.